apps/app/github-environment.tf

```diff
@@ -32,7 +32,7 @@ resource "github_branch_protection" "deployment" {
 
   required_status_checks {
     strict   = var.accesses_real_data
-    contexts = ["Lint"]
+    contexts = ["Semgrep CI"]
```

Comment:
(if you take my minor feedback on your other PR then the name here will change)

Reply:
```suggestion
    contexts = ["Code Scanning"]
```